.github/workflows/windows.yml

```diff
@@ -123,3 +123,58 @@ jobs:
       - name: Test
         run: |
           python dev.py test -j2 --mode full
+
+
+  #############################################################################
+  full_build_sdist_wheel:
+    # Python 3.11
+    # python -m build && pip install dist/scipy*.whl
+    # 'full' test run,
+    # -Duse-pythran=false
+    # TODO: enable ILP64 once possible
+    name: cp311 (build sdist + wheel), full, no pythran
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: windows-2019
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+          cache: 'pip'
+          cache-dependency-path: 'environment.yml'
+
+      - name: Win_amd64 - install rtools
+        run: |
+          # mingw-w64
+          choco install rtools --no-progress
+          echo "c:\rtools40\ucrt64\bin;" >> $env:GITHUB_PATH
+
+      - name: Install OpenBLAS
+        shell: bash
+        run: |
+          set -xe
+          bash tools/wheels/cibw_before_build_win.sh .
+          echo "PKG_CONFIG_PATH=c:\opt\64\lib\pkgconfig;" >> $GITHUB_ENV
+
+      - name: pip-packages
+        run: |
+          python -m pip install build delvewheel numpy cython pybind11 meson-python meson ninja pytest pytest-xdist pytest-timeout pooch rich_click click doit pydevtool
+
+      - name: Build
+        run: |
+          python -m build -Csetup-args="-Duse-pythran=false"
+
+          $env:wheel_name=Get-ChildItem -Path dist/* -Include *.whl
+          delvewheel repair --add-path c:\opt\openblas\openblas_dll -w dist $env:wheel_name
+          
+          python -m pip install --only-binary=scipy --no-index --find-links=dist scipy
+
+      - name: Test
+        run: |
+          cd $RUNNER_TEMP
+          pytest -k test --pyargs scipy
```

Comment:
`-k test` seems unnecessary, it's a regex that selects every single test. Running the full test suite like this works; you may want to append a `  # runs full test suite` comment (`pytest --pyargs scipy` has the opposite default as `dev.py` and `scipy.test()`, which may not be obvious to everyone).